app/controllers/RepaymentAmountSummaryController.scala

```diff
@@ -49,9 +56,16 @@ class RepaymentAmountSummaryController @Inject() (
       Ok(view(sections, backLink(request.userAnswers)))
   }
 
-  def onSubmit(): Action[AnyContent] = (identify andThen getData andThen requireData) {
+  def onSubmit(): Action[AnyContent] = (identify andThen getData andThen requireData).async {
     implicit request =>
-      Redirect(nextPage(request.userAnswers))
+      if (request.userAnswers.dutyTypeTaxDetails.totalClaim < appConfig.allowCmaThresholds.reclaimTotal)
```

Comment:
RED: This resetting of payment type is only done from POST on the summary page.  I was able to go from CYA to the "What do you want to reclaim?" Uncheck two claim types and click continue (dropping my value to £10.01)  then using the in page < Back navigator to go back to the CYA page and still have CMA as a the payment option.